scipy/stats/tests/test_distributions.py

```diff
@@ -5297,6 +5297,19 @@ def test_sf_isf(self):
         x1 = stats.nakagami.isf(sf, nu)
         assert_allclose(x1, x0, rtol=1e-13)
 
+    @pytest.mark.parametrize('nu', [1.6, 2.5, 3.9])
+    @pytest.mark.parametrize('loc', [25.0, 10, 35])
+    @pytest.mark.parametrize('scale', [13, 5, 20])
+    def test_fit(self, nu, loc, scale):
+        # The first tuple of the parameters' values is discussed in gh-10908
```

Comment:
Are the rest arbitrary, or do they currently fail on master?

Reply:
21 or those 27 combinations fail on master.

From the experimentations I've done with data generation, a great number of cases (easily more than half of them) show that the computations of the ML estimators for Nakagami is not stable. This motivated my original and more exhaustive parametrisation of this test.

Reply:
Gotcha. Thanks!